ci/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM fedora
+MAINTAINER Igor Gnatenko <i.gnatenko.brain@gmail.com>
+
+WORKDIR /opt/rpm
+COPY . .
+
+RUN echo -e "deltarpm=0\ninstall_weak_deps=0\ntsflags=nodocs" >> /etc/dnf/dnf.conf
+RUN dnf -y update
+RUN dnf -y install \
```

Comment:
Which calls into this: https://github.com/projectatomic/rpm-ostree/blob/f25444554dac95c5ff469f64192bad17a2916c3a/ci/libbuild.sh#L18

(Which is a good example of a minor annoyance from the yum → dnf rename)